<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDataMainInfoAreaComponent</class>
 <widget class="QMainWindow" name="CDataMainInfoAreaComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>401</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="qw_centralEmptyWidgetUsedAsPadding">
   <property name="maximumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="swift::gui::CDockWidgetInfoArea" name="dwp_Mapping">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Mapping editor</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="qw_MappingOuter">
    <layout class="QVBoxLayout" name="vl_MappingOuter">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="fr_MappingInner">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="vl_MappingInner">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="swift::gui::components::CDbMappingComponent" name="comp_Mapping">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="swift::gui::CDockWidgetInfoArea" name="dwp_DataInfoArea">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Database data</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="qw_DataInfoAreaOuter">
    <layout class="QVBoxLayout" name="vl_AircraftModelsOuter">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="fr_DataInfoAreaInner">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <layout class="QVBoxLayout" name="vl_AircraftModelsInner">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="swift::gui::components::CDataInfoAreaComponent" name="comp_DataInfoArea" native="true"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="swift::gui::CDockWidgetInfoArea" name="dwp_Settings">
   <property name="allowedAreas">
    <set>Qt::TopDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Settings and login</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="qw_SettingsOuter">
    <layout class="QVBoxLayout" name="vl_SettingsOuter">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="fr_SettingsInner">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="vl_SettingsInner">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="swift::gui::components::CDataSettingsComponent" name="comp_Settings">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="swift::gui::CDockWidgetInfoArea" name="dwp_Log">
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>400</height>
    </size>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Logging</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="qw_LogOuter">
    <layout class="QVBoxLayout" name="vl_LogOuter">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="fr_LogInner">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="vl_LogInner">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="swift::gui::components::CLogComponent" name="comp_Log">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swift::gui::CDockWidgetInfoArea</class>
   <extends>QDockWidget</extends>
   <header>gui/dockwidgetinfoarea.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CLogComponent</class>
   <extends>QFrame</extends>
   <header>gui/components/logcomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CDataInfoAreaComponent</class>
   <extends>QWidget</extends>
   <header>gui/components/datainfoareacomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CDbMappingComponent</class>
   <extends>QFrame</extends>
   <header>gui/components/dbmappingcomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CDataSettingsComponent</class>
   <extends>QFrame</extends>
   <header>gui/components/datasettingscomponent.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
